src/components/NcHeaderMenu/NcHeaderMenu.vue

```diff
@@ -69,6 +69,8 @@ import { getTrapStack } from '../../utils/focusTrap.js'
 import NcButton from '../NcButton/index.ts'
 
 const {
+	ariaLabel = undefined,
+	description = undefined,
```

Comment:
Doesn't it have `undefined` value anyway when it is not set?

Reply:
Yes but this is a bit more explicit when reading and also satisfies https://eslint.vuejs.org/rules/require-default-prop

Yes we can ignore but 1 comment line to ignore or just add this default is no difference and I consider this here cleaner. (Also after script-setup compilation its no overhead).

Reply:
I forgot about the eslint rule